src/main/scala/GameHelpers.scala

```diff
@@ -0,0 +1,69 @@
+import scala.util.Random
+
+object GameHelpers {
+
+  def fillRack(rack: List[Tile], letterBag: List[Tile]): (List[Tile], List[Tile]) = {
+    val (drawnTiles, newBag) = letterBag.splitAt(7 - rack.length)
+    val filledRack = rack ++ drawnTiles
+
+    if (filledRack.length < 7) {
+      println("\u001b[0m---------")
+      print("\u001b[31;1mWARNING: ")
+      println("\u001b[36mThe letter bag is empty.")
+      print("\u001b[0m")
+    }
+
+    (filledRack, newBag)
+  }
+
+  def removeTilesFromRack(word: String, rack: List[Tile]): List[Tile] = {
+    val wordTiles: List[Tile] = word.toList.map(c => Tile(c))
+    rack diff wordTiles
+  }
+
+  def calculateWordScore(word: String): Int = {
+    word.toList.map(Tile(_).value).sum
+  }
+
+  def isInDictionary(word: String): Boolean = {
+    if (Dictionary.all.contains(word.toLowerCase())) true else false
```

Comment:
Any time you see a statement returning `true` or `false` ask whether you need to use these literals. What would happen if you just returned
```scala
Dictionary.all.contains(word.toLowerCase)
```
?